plugins/woocommerce/tests/php/src/Internal/DataStores/Orders/OrdersTableQueryTests.php

```diff
@@ -1,9 +1,12 @@
 <?php
 
+use Automattic\WooCommerce\Internal\DataStores\Orders\OrdersTableQuery;
 use Automattic\WooCommerce\RestApi\UnitTests\Helpers\OrderHelper;
 use Automattic\WooCommerce\RestApi\UnitTests\HPOSToggleTrait;
 use Automattic\WooCommerce\Utilities\OrderUtil;
 
+require_once __DIR__ . '/../../../../helpers/HPOSToggleTrait.php';
```

Comment:
Is this necessary? It doesn't look like you're using the toggle in the new tests. I commented it out and ran the tests again and there were no errors...

Reply:
I needed to add this in order to be able to run the tests in my local setup, otherwise I would get a "trait not found" error (maybe I have something misconfigured); but indeed it was intended as a temporary addition. I'll remove it.

Reply:
> So it seems like maybe_remap_meta_args needs to be able to operate recursively

Adding a note that WordPress offers a recursive [`map_deep` function](https://developer.wordpress.org/reference/functions/map_deep/) that may be useful here.